pkg/clustermanager/configs.go

```diff
@@ -9,11 +9,15 @@ import (
 func GenerateMasterConfiguration(masterNode Node, masterNodes []Node, etcdNodes []Node) string {
 	masterConfigTpl := `apiVersion: kubeadm.k8s.io/v1beta1
 kind: ClusterConfiguration
+kubernetesVersion: stable
```

Comment:
CC is crying on this one, this makes the function quite big. Maybe moving the template contents into a variable, instead of having it inside the function? WDYT?

Reply:
IMHO is not a good solution since we move the test less comprehensible, in this case I'll prefer ignore CC issue, but if you think still useful I'll move this evening.